src/core/Form/InputClearButton/InputClearButton.baseStyles.ts

```diff
@@ -25,4 +25,16 @@ export const baseStyles = (theme: SuomifiTheme) => css`
       }
     }
   }
+
+  &[disabled],
+  &:disabled {
+    cursor: not-allowed;
+    &.fi-input-clear-button {
+      & .fi-input-clear-button_icon {
+        & .fi-icon-base-fill {
+          fill: ${theme.colors.depthBase};
+        }
+      }
+    }
+  }
```

Comment:
Same as with toggle button, I'll suggest that we move this inside of the so called base class, to `fi-input-clear-button` that is few lines above.

Reply:
See my comment above

Reply:
Changed this as well based on comments above